### DISCUSSION TITLE
Regression for many short sequences

### DISCUSSION THREAD

**Discussion by nils-fl** [alan-turing-institute/sktime#2085] [Q&A]
Hi,

I have the following problem:
I have a lot (>100.000) of sequences of two or more parameters that could indicate if an event happens a number of times.
Previously I successfully used Rocket + RidgeClassifier to classify IF an event happens on these sequences.
Now I want to go a step further and check if I can predict the number of events.

Is there a model, that I am overseeing, that could help conduct a regression for the multivariate sequences?

Thank you!

EDIT:
That's probably a stupid question, I could just use Rocket + Ridge instead of the classifier?

EDIT:
Yes, it was that simple. 
Sorry about that, I'll leave my comment like that, just in case someone else thinks about the same on a Friday evening - too exhausted to think clearly.

**Comment by fkiraly** [comment]
Yes, that pipeline sounds like the closest for TSR, great that the question answered itself in the end :-)
